micro-deps/micro-deps-spring-config/src/main/java/com/ofg/infrastructure/discovery/ZookeeperServiceResolverConfiguration.java

```diff
@@ -1,75 +1,53 @@
 import org.springframework.cloud.zookeeper.ZookeeperAutoConfiguration;
+import org.springframework.cloud.zookeeper.discovery.ZookeeperDiscoveryAutoConfiguration;
+import org.springframework.cloud.zookeeper.discovery.ZookeeperDiscoveryClientConfiguration;
 import org.springframework.cloud.zookeeper.discovery.ZookeeperDiscoveryProperties;
-import org.springframework.cloud.zookeeper.discovery.ZookeeperInstance;
-import org.springframework.cloud.zookeeper.discovery.ZookeeperServiceDiscovery;
 import org.springframework.cloud.zookeeper.discovery.dependency.ZookeeperDependencies;
+import org.springframework.cloud.zookeeper.serviceregistry.ZookeeperAutoServiceRegistrationAutoConfiguration;
+import org.springframework.cloud.zookeeper.serviceregistry.ZookeeperServiceRegistry;
+import org.springframework.cloud.zookeeper.serviceregistry.ZookeeperServiceRegistryAutoConfiguration;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
 import org.springframework.context.annotation.Profile;
 import org.springframework.core.env.Environment;
-import org.springframework.util.SocketUtils;
 
 /**
- * Configuration that binds together whole service discovery. Imports:
- * <p/>
- * <ul>
- * <li>{@link AddressProviderConfiguration} - contains beans related to microservice's address and port resolution</li>
- * <li>{@link ServiceDiscoveryInfrastructureConfiguration} - contains beans related to connection to service discovery provider (available only in {@link BasicProfiles#PRODUCTION}</li>
- * <li>{@link DependencyResolutionConfiguration} - Configuration of microservice's dependencies resolving classes.
- * </ul>
+ * Configuration that binds together whole service discovery.
  */
-@Import(ConsumerDrivenContractConfiguration.class) 
-@ImportAutoConfiguration({SpringCloudZookeeperDiscoveryClientConfiguration.class, ZookeeperAutoConfiguration.class, UtilAutoConfiguration.class})
+@Import(ConsumerDrivenContractConfiguration.class)
+@ImportAutoConfiguration({
+        ZookeeperDiscoveryClientConfiguration.class,
+        ZookeeperDiscoveryAutoConfiguration.class,
+        ZookeeperServiceRegistryAutoConfiguration.class,
+        ZookeeperAutoServiceRegistrationAutoConfiguration.class,
+        ZookeeperAutoConfiguration.class,
+        UtilAutoConfiguration.class
+})
 @Configuration
 @Profile(BasicProfiles.SPRING_CLOUD)
 public class ZookeeperServiceResolverConfiguration {
 
-    static final Integer DEFAULT_SERVER_PORT = 8080;
-
     @Autowired Environment environment;
     @Autowired ApplicationContext applicationContext;
 
-    @Bean(initMethod = "build")
-    public ZookeeperServiceDiscovery zookeeperServiceDiscovery(CuratorFramework curator, ZookeeperDiscoveryProperties zookeeperDiscoveryProperties, InstanceSerializer<ZookeeperInstance> instanceSerializer) {
```

Comment:
`ZookeeperServiceDiscovery` is deprecated. `ZookeeperServiceRegistry` used instead.